File: express/app/src/main/java/com/changsukuaidi/www/modules/login/launcher/LauncherModule.java
package com.changsukuaidi.www.modules.login.launcher;

import dagger.Module;
import dagger.Provides;

/**
 * @Describe
 * @Author zhouhao
 * @Date 2018/2/7
 * @Contact 605626708@qq.com
 */

@Module
class LauncherModule {
    private final LauncherFragment mView;

    LauncherModule(LauncherFragment view) {
        mView = view;
    }

    @Provides
    LauncherFragment provideLoginView(){
        return mView;
    }
}

File: WIFIDirect/app/src/main/java/com/thc/www/p2p/module/LauncherModule.kt
package com.thc.www.p2p.module

import com.thc.www.p2p.android.launcher.LauncherFragment
import com.thc.www.p2p.annotation.FragmentScope
import dagger.Module
import dagger.Provides

/**
 * @Describe
 * @Author thc
 * @Date  2018/5/7
 */
@Module
class LauncherModule(private val view: LauncherFragment) {
    @FragmentScope
    @Provides
    fun provideView() = view
}